protocol/chainlib/common.go

```diff
@@ -30,6 +30,7 @@ const (
 	relayMsgLogMaxChars        = 200
 	RPCProviderNodeAddressHash = "Lava-Provider-Node-Address-Hash"
 	RPCProviderNodeExtension   = "Lava-Provider-Node-Extension"
+	RpcProviderLoadRateHeader  = "Lava-Provider-load-rate"
```

Comment:
Please follow the letter case other headers
```suggestion
	RpcProviderLoadRateHeader  = "Lava-Provider-Load-Rate"
```